x/shared/helpers/service_configs.go

```diff
@@ -68,7 +68,9 @@ func ValidateSupplierServiceConfigs(services []*sharedtypes.SupplierServiceConfi
 				return fmt.Errorf("endpoint.RpcType is not a valid RPCType: %v", serviceConfig)
 			}
 
-			// TODO_BLOCKER(@okdas): Validate configs once they are being used
+			// TODO_MAINNET(@okdas)/TODO_DISCUSS: Either add validation for `endpoint.Configs` (can be a part of
+			// `parseEndpointConfigs`), or change the config structure to be more clear about what is expected here
+			// as currently, this is just a map[string]string, when values can be other types.
```

Comment:
cc @red-0ne

Reply:
@bryanchriswhite had a chat with @red-0ne [in Discord](https://discord.com/channels/824324475256438814/1138895490331705354/1260969507456024596) so I'll adjust the comment accordingly.